src/app/admin/modals/user-settings-modal/user-settings-modal.component.html

```diff
@@ -0,0 +1,112 @@
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.first_name" type="input" class="form-control"
+          placeholder="Name" />
+      </div>
+    </div>
+    <div class="form-group" ng-required="isNew">
+      <label class="col-sm-3 control-label">Last Name</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.last_name" type="input" class="form-control" placeholder="Surname" />
+      </div>
+    </div>
+    <div class="form-group" ng-required="isNew">
+      <label class="col-sm-3 control-label">Preferred Name</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.nickname" type="input" class="form-control" placeholder="Name" />
+      </div>
+    </div>
+    <div class="form-group" ng-required="isNew">
+      <label class="col-sm-3 control-label">Email</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.email" type="email" class="form-control" placeholder="name@university.edu" />
+      </div>
+    </div>
+    <hr />
+    <div class="form-group" *ngIf="user.system_role === 'Student'">
+      <label class="col-sm-3 control-label">Student ID</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.student_id" type="input" class="form-control" placeholder="1744070" />
+        <p class="help-block">Please ensure your institution's student identification number matches this value</p>
+      </div>
+    </div>
+    <div class="form-group">
+      <label class="col-sm-3 control-label">Avatar</label>
+      <div class="col-sm-7">
+        <user-icon [size]="50" [user]="user" style="max-width: 50px; max-height: 50px; font-size: 18px"></user-icon>
+        <a href="http://en.gravatar.com/support/activating-your-account/" target="_blank"
+          class="btn btn-sm btn-default pull-right">
+          Change Avatar
+        </a>
+        <p class="help-block">
+          Sign up to
+          <a href="http://en.gravatar.com/support/activating-your-account/" target="_blank">Gravatar</a> and associate
+          your email above to your account to give yourself an avatar
+        </p>
+      </div>
+    </div>
+    <div class="form-group" if-role="Admin Convenor" ng-required="isNew">
+      <label class="col-sm-3 control-label">System Role</label>
+      <div class="col-sm-7">
+        <mat-radio-group
+          [(ngModel)]="user.system_role"
+          class="user-role">
+          <mat-radio-button value="Administrator">
+            Administrator
+          </mat-radio-button>
+          <mat-radio-button value="Convenor">
+            Convenor
+          </mat-radio-button>
+          <mat-radio-button value="Tutor">
+            Tutor
+          </mat-radio-button>
+          <mat-radio-button value="Student">
+            Student
+          </mat-radio-button>
+        </mat-radio-group>
+        <p *ngIf="user.id === CurrentUser.id" class="help-block text-center">
+          You cannot modify your own permissions
+        </p>
+      </div>
+    </div>
```

Comment:
This section does not work due to `ERROR TypeError: Cannot read properties of undefined (reading 'first_name')`